setup.py

```diff
@@ -115,28 +115,45 @@ def run(self):
         version=version,
         author='Edgar Riba',
         author_email='contact@kornia.org',
-        url='https://github.com/kornia/kornia',
+        url='https://www.kornia.org',
+        download_url='https://github.com/kornia/kornia',
+        license='Apache License 2.0',
         description='Open Source Differentiable Computer Vision Library for PyTorch',
         long_description=long_description,
         long_description_content_type='text/markdown',
-        license='Apache License 2.0',
         python_requires='>=3.6',
-        # Test
         setup_requires=['pytest-runner'],
         tests_require=['pytest'],
-        # Package info
         packages=find_packages(exclude=('docs', 'test', 'examples')),
         package_data={"kornia": ["py.typed"]},
         zip_safe=True,
         install_requires=requirements,
+        keywords=['computer vision', 'deep learning', 'pytorch']
+        project_urls={
+            "Bug Tracker": "https://github.com/kornia/kornia/issues",
+            "Documentation": "https://kornia.readthedocs.io/en/latest/",
```

Comment:
Just to be more consistent XD.
```suggestion
            "Documentation": "https://kornia.readthedocs.io/en/latest",
```